### ISSUE TITLE
Can't create collections for new partner ddr-phljacl

### ISSUE THREAD

**Issue by gjost** [denshoproject/ddr-local#335]
sara reports this error from Celery when trying to create a new collection:
```
Traceback (most recent call last):
  File "/opt/ddr-local/venv/ddrlocal/lib/python3.9/site-packages/celery/app/trace.py", line 543, in trace_task
    task_after_return(
  File "/opt/ddr-local/ddrlocal/webui/tasks/collection.py", line 120, in after_return
    collection_id = retval['collection_id']
TypeError: 'Exception' object is not subscriptable
```

**Comment by GeoffFroh** [comment]
The `ddr-phljacl` partner is brand-new. It is possible that we missed a step in the setup. See the process we followed here: https://app.gitbook.com/o/Wu8cMjykvVCCuZxUaR0g/s/2qWokgjY6AXnKVQ7X64v/ddr-admin-workflows/adding-a-partner-organization

**Comment by gjost** [comment]
I got the `TypeError: 'Exception' object is not subscriptable` when attempting to create a new `ddr-testing` collection, so I don't think it's limited to `ddr-phljacl`

**Comment by gjost** [comment]
On Thursday we were able to use the web UI to create a new `ddr-phljacl-6` without errors.  Don't know what changed but it worked.  Sara wanted to start over with `ddr-phljacl-1` so I removed the existing IDs from `ddridservice`, removed the existing `ddr-phljacl-*` repos from `mits3` and from `/media/qnfs/kinkura/gold/`.

Looks like it worked?  I'm not sure what changed, which worries me, but I don't see anything wrong now.
